### DISCUSSION TITLE
Edge Function Wall Clock Limit Exceeded

### DISCUSSION THREAD

**Discussion by justinekatebe** [supabase/supabase#17206] [Questions]
Hey guys, I recently started getting this warning telling me "Wall clock time exceeded" in my edge function's logs. 

How do I make it go away?

<img width="543" alt="Screenshot 2023-09-06 at 06 25 26" src="https://github.com/supabase/supabase/assets/82674078/352de1e6-e619-407c-a690-9751bd607fb1">

**Comment by danielstaleiny** [comment]
bumb, I also see rapid increase in silently failed edge requests caused by wall clock time exceeded. We are using edge functions for some background tasks and some of them has to wait for fetch 30-60seconds to answer, which results in these warnings and resulting in not finishing the requests.

**Reply by justinekatebe** [reply]
Yep, mine too. My edge functions are called by an every minute cron job and sometimes they don't call the edge functions at all. It's now getting frustrating.

**Reply by danielstaleiny** [reply]
100% agreed, I am forced to migrate to other solution because of this. I can also confirm that it only happened to be problem 1week ago~.

**Reply by ffd114** [reply]
Also happened to me, I ask their support and they said that it will be configurable per project in the future. Hopefully they will get it done. My workaround right now, since I use Nuxt only, I just use server handlers to handle the long background tasks

**Comment by verissimor** [comment]
when using ai models the timeouts are reached quite easily. Have anyone found a workaround?

**Reply by danielstaleiny** [reply]
yes, use server and not edge function.
